packages/strapi-design-system/src/Link/Link.js

```diff
@@ -12,18 +12,34 @@ const LinkWrapper = styled.a`
   align-items: center;
   text-decoration: none;
   pointer-events: ${({ disabled }) => (disabled ? 'none' : undefined)};
+
+  ${Typography} {
+    transition: color 150ms ease-out;
+  }
+
   svg path {
     fill: ${({ disabled, theme }) => (disabled ? theme.colors.neutral600 : theme.colors.primary600)};
   }
+
   svg {
     font-size: ${10 / 16}rem;
   }
-  &:hover,
+
+  &:hover {
+    ${Typography} {
+      color: ${({ theme }) => theme.colors.primary500};
```

Comment:
should it not impact the icons too? 👀

<img width="116" alt="image" src="https://user-images.githubusercontent.com/71838159/194279749-de439019-f542-416e-9d14-f29361e17f3b.png">
<img width="347" alt="image" src="https://user-images.githubusercontent.com/71838159/194279990-e1f75676-3403-41b2-a15c-f237826e1600.png">

Reply:
That would be nice indeed if feasible

Reply:
I wasn't sure and thanks both for letting me know, i'll make those changes 😎

Reply:
You also may want to add `:focus,`?